service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseManagementOrderService.java

```diff
@@ -20,33 +18,18 @@
 import static uk.gov.hmcts.reform.fpl.enums.CaseManagementOrderKeys.ORDER_ACTION;
 import static uk.gov.hmcts.reform.fpl.enums.CaseManagementOrderKeys.RECITALS;
 import static uk.gov.hmcts.reform.fpl.enums.CaseManagementOrderKeys.SCHEDULE;
-import static uk.gov.hmcts.reform.fpl.model.HearingDateDynamicElement.getHearingDynamicElement;
-import static uk.gov.hmcts.reform.fpl.model.common.DocumentReference.buildFromDocument;
+import static uk.gov.hmcts.reform.fpl.enums.DocmosisTemplates.CMO;
 import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.DATE;
 import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.parseLocalDateFromStringUsingFormat;
 
 //TODO: this class will take some of the methods out of draftCMO service. FPLA-1479
 @Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class CaseManagementOrderService {
     private final Time time;
-    private final HearingBookingService hearingBookingService;
-
-    public CaseManagementOrderService(Time time, HearingBookingService hearingBookingService) {
-        this.time = time;
-        this.hearingBookingService = hearingBookingService;
-    }
-
-    public CaseManagementOrder addDocument(CaseManagementOrder caseManagementOrder, Document document) {
-        return caseManagementOrder.toBuilder()
-            .orderDoc(buildFromDocument(document))
-            .build();
-    }
-
-    public CaseManagementOrder addAction(CaseManagementOrder order, OrderAction orderAction) {
-        return order.toBuilder()
-            .action(orderAction)
-            .build();
-    }
+    private final DraftCMOService draftCMOService;
```

Comment:
I don't think this service should be dependant on the draft service, thoughts on moving the required methods into this one?

Reply:
I totally agree, I don't wanna start hammering out loads of files changes when I'm no longer touching the about to submit of the controller though. What do you reckon?

Reply:
sure thing, new PR it is